### PR TITLE
Archeo: Add index and search templates

### LINKED ISSUE #5425: Archeo: Templates: Page templates - Search

### PR DESCRIPTION
**Pull request by scruffian**
#### Changes proposed in this Pull Request:
Adds and index and search template for Archeo:

Screenshot 2022-02-09 at 13 14 35
Screenshot 2022-02-09 at 12 55 00
Screenshot 2022-02-09 at 14 27 58

#### Related issue(s):
Fixes #5425 and #5424 and #5427

### CODE REVIEW

File: archeo/templates/search.html [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+<main class="wp-block-query">
+	<!-- wp:search {"label":"Search","showLabel":false,"buttonText":"Search"} /-->
+	<!-- wp:spacer {"height":40} -->
+	<div style="height:40px" aria-hidden="true" class="wp-block-spacer"></div>
+	<!-- /wp:spacer -->
+	<!-- wp:query-title {"type":"archive","style":{"spacing":{"margin":{"bottom":"60px"}}}} /-->
+	<!-- wp:post-template -->
+	<!-- wp:template-part {"slug":"query"} /-->
+	<!-- /wp:post-template -->
+	<!-- wp:template-part {"slug":"pagination"} /-->
```
Comment (jffng) [inline archeo/templates/search.html]: The pagination is not working for me. It shows the same posts on every page (e.g. `/?query-page-2`).

I switched to Livro and it's working fine. I wonder if it's related to putting the query and / or pagination inside a template part?

**Review by jffng — COMMENTED** [review]
Can't comment on the design because there's no comp, but I had one issue about the functionality of the query + pagination.

File: archeo/templates/search.html (hunk 1 above)
Comment (scruffian) [reply archeo/templates/search.html]: It seems to happen when the pagination is in a different template, so I've removed that change for now. I created an issue for it here: https://github.com/WordPress/gutenberg/issues/38684

**Comment by kjellr** [comment]
This looks good. Two small changes and we're good to go from a design perspective:

- There's currently a 40px spacer below each post in the query. Let's change that to 60px. 
- The archive title has 60px of bottom padding currently. Let's change that to 80px.

Also, I'm seeing this text cut off: 
Screen Shot 2022-02-10 at 10 42 07 AM

I think that text is too large for mobile anyway... I'll open another PR to tidy up these sizes. (_Edit:_ Done in https://github.com/Automattic/themes/pull/5522)

**Comment by jffng** [comment]
> Two small changes and we're good to go from a design perspective 

Done in [511f4a1](https://github.com/Automattic/themes/pull/5489/commits/511f4a1e65bd99c8baf8680d9757286504ddf774).

**Review by jffng — APPROVED** [review]

**Pull request merged**